spec/factories/projects.rb

```diff
@@ -3,7 +3,9 @@
 FactoryGirl.define do
   factory :project do
     title "MyString"
-    logo "MyString"
+    project_key "MyString"
```

Comment:
This should probably use a generator so it makes a unique string each time it is called. 
Here is an example:
https://github.com/concord-consortium/lara/blob/526c2f0e382b7561ca883cba8672d0c3066882ce/spec/factories/plugins.rb#L4-L10

The sequence call creates a generator named `:word` that then returns a string. It takes the `n` param. So in this case you could call it `:project_key` and return something like "test-key-#{n}". Then you use it in the factory with `project_key { generate(:project_key) }`

Reply:
Fix looks good.